pkg/api/types.go

```diff
@@ -252,7 +252,7 @@ type PersistentVolumeSpec struct {
 	PersistentVolumeSource `json:",inline"`
 	// AccessModes contains all ways the volume can be mounted
 	AccessModes []AccessModeType `json:"accessModes,omitempty"`
-	// holds the binding reference to a PersistentVolumeClaim
+	// ClaimRef is a non-binding reference to the claim bound to this volume
```

Comment:
This comment could be more descriptive.  Specifically explain what non-binding means and the bi-directionality of the linkage.